### PR TITLE
add smtp password to install page

### PR DESCRIPTION
**Pull request by 21h**
get smtp password from  GITEA__mailer__PASSWD

### CODE REVIEW

File: routers/install/install.go [hunk 1]
```diff
@@ -137,7 +137,8 @@ func Install(ctx *context.Context) {
 		form.SMTPHost = setting.MailService.Host
 		form.SMTPFrom = setting.MailService.From
 		form.SMTPUser = setting.MailService.User
-	}
+		form.SMTPPasswd = setting.MailService.Passwd
+}
```
Comment (zeripath) [inline routers/install/install.go]:
```suggestion
	}
```

**Comment by zeripath** [comment]
hmm... 

three thoughts:

1. Given we're blandly showing the DBPasswd above I think the sec concerns about showing the SMTPPasswd on the install page can be ignored in this PR - but if you have any thoughts about handling that concern that would be useful.
2. We're getting a lot of PRs like this -  Are there other settings that would be useful? If you think there's another one you might want can we just set that now.
3. We need to refactor our config to just make this a lot more automatic...

**Review by zeripath — COMMENTED** [review]
otherwise LGTM.

**Comment by 21h** [comment]
security on install page?) 
to be more automatic:
0) create admin user with creds from ENVs
1) automatic installation without asking to press install button
2) completely turn off ssh access at installation
3) set gitea url fron ENVs, at this moment it generated from hostname and port

**Comment by wxiaoguang** [comment]
@zeripath Continue to merge, or close?

**Review by wxiaoguang — APPROVED** [review]
Since there is already an password input on the installation page, and no security problem, so no problem at the moment.


If this shouldn't be merged, then it could be closed with reasons.

**Review by zeripath — APPROVED** [review]

**Comment by lunny** [comment]
make L-G-T-M work

**Comment by 21h** [comment]
наконец-то мой коммит приняли

**Pull request merged**